bindings/python/iota_sdk/secret_manager/secret_manager.py

```diff
@@ -139,7 +139,64 @@ def generate_addresses(self,
 
         options = humps.camelize(options)
 
-        return self._call_method('generateAddresses', {
+        return self._call_method('generateEd25519Addresses', {
+            'options': options
+        })
+
+    def generate_evm_addresses(self,
+                           account_index=None,
+                           start=None,
+                           end=None,
+                           internal=None,
+                           coin_type=None,
+                           ledger_nano_prompt=None):
+        """Generate EVM addresses.
+
+        Parameters
+        ----------
+        secret_manager : Any type of SecretManager.
+            The secret manager to use. Can be (MnemonicSecretManager, SeedSecretManager, StrongholdSecretManager or LedgerNanoSecretManager.
```

Comment:
```suggestion
```

Reply:
Isn't that `self`?

Reply:
self is the SecretManager class, but that doesn't get provided by the user to call this function (it's also wrong in generate_ed25519_addresses)

Reply:
okay I'll remove both then